src/com/xilinx/rapidwright/design/tools/RelocationTools.java

```diff
@@ -0,0 +1,101 @@
+package com.xilinx.rapidwright.design.tools;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+
+import com.xilinx.rapidwright.design.*;
```

Comment:
One thing we avoid is wildcard imports as it can impact startup and runtime performance.  IDEs should automatically handle the imports for you so it shouldn't be any extra work to import explicitly.  If you find that it is cumbersome, let me know.